components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIConsumer.java

```diff
@@ -319,7 +319,7 @@ public void addComment(APIIdentifier identifier, String comment,
      * @param userId      User Id
      * @throws APIManagementException if failed to add Application
      */
-    public String addApplication(Application application, String userId) throws APIManagementException;
+    public int addApplication(Application application, String userId) throws APIManagementException;
```

Comment:
Fixed in #1581(https://github.com/wso2/carbon-apimgt/commit/73c207a285e09dde2475ebe8182da3532a01431c)